src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/ExecutionPolicies/HmrcExecutionPolicy.cs

```diff
@@ -37,7 +37,7 @@ protected override T OnException<T>(Exception ex)
             {
                 _logger.Error(ex, $"Exceeded retry limit - {ex.Message}");
             }
-            return default(T);
+            throw ex;

```

Comment:
Am i correct in saying that if you throw an error when you can not find an english fraction for that paye scheme then you can not add a paye scheme which has no english fraction. (application will always redirect you to There's been an error page and refuse to add the paye scheme)

Reply:
Nope, the paye scheme is added and then a message is pushed on to a queue to get the English Fraction and Levy for the scheme as a background task.  If that process fails the user would be unaware and the scheme would still be added to their account.